src/npctrade.h

```diff
@@ -30,12 +30,18 @@ struct item_pricing {
     bool selected;
 };
 
+namespace npc_trading
+{
+
+bool pay_npc( npc &np, int cost );
+
 int cash_to_favor( const npc &, int cash );
 
 inventory inventory_exchange( inventory &inv,
                               const std::set<item *> &without, const std::vector<item *> &added );
 std::vector<item_pricing> init_selling( npc &p );
 std::vector<item_pricing> init_buying( npc &p, player &u );
 bool trade( npc &p, int cost, const std::string &deal );
+};
```

Comment:
Pedantic warning about unnecessary ;
```suggestion
}
```